README.md

```diff
@@ -404,3 +432,15 @@ Cron check-ins are configured in the following locations:
 2. `config/settings.yml`: Stubs out a check-in key for each cron job. Since we may not want to have a check-in for all environments, this stub key will be used and produce a null check-in.
 3. `config/settings/production.yml` in shared_configs: This contains the actual check-in keys.
 4. HB notification page: Check-ins are configured per project in HB. To configure a check-in, the cron schedule will be needed, which can be found with `bundle exec whenever`. After a check-in is created, the check-in key will be available. (If the URL is `https://api.honeybadger.io/v1/check_in/rkIdpB` then the check-in key will be `rkIdpB`).
+
+## prescat doc updates
+
+Met w/ suntzu:
+
+- identify errored druids with sql
+  - paragraph with some code
+  - resque
+- update sql in db readme 
+- report to HB
+- export of all replication audit errors from HB, parse out druid
+- 
```

Comment:
I'm guessing this isn't ready for review yet because this section looks like an outline.   Might I suggest a draft PR until you are ready for it to be reviewed?

Reply:
Whoops I wondered where those notes went! Thanks I'm removing :-) 

There may have been differing opinions about where this text belonged. My opinion was that it belonged in the README.